txscript/script.go

```diff
@@ -303,11 +327,13 @@ func removeOpcodeByData(script []byte, dataToRemove []byte) []byte {
 // being spent, in addition to the final transaction fee. In the case the
 // wallet if fed an invalid input amount, the real sighash will differ causing
 // the produced signature to be invalid.
-func calcWitnessSignatureHashRaw(scriptSig []byte, sigHashes *TxSigHashes,
+func calcWitnessSignatureHashRaw(subScript []byte, sigHashes *TxSigHashes,
```

Comment:
Nice to fix this naming confusion! I always tripped over this, since we set the pub key hash here in case of P2WPKH...